src/modifiers/ComparisonModifier.js

```diff
@@ -76,6 +76,36 @@ class ComparisonModifier extends Modifier {
     return `${this.comparePoint || ''}`;
   }
 
+  /* eslint-disable class-methods-use-this */
+  /**
+   * Empty default compare point definition
+   *
+   * @param {StandardDice|RollGroup} _context The object that the modifier is attached to
+   *
+   * @returns {null}
+   */
+  defaultComparePoint(_context) {
+    return null;
+  }
+  /* eslint-enable class-methods-use-this */
+
+  /**
+   * Eases processing of simple "compare point only" defaults
+   *
+   * @param {StandardDice|RollGroup} _context The object that the modifier is attached to
+   *
+   * @returns {object}
+   */
+  defaults(_context) {
+    const comparePointConfig = this.defaultComparePoint(_context);
+
+    if (comparePointConfig) {
+      return { comparePoint: new ComparePoint(...this.defaultComparePoint(_context)) };
```

Comment:
This could use the `comparePointConfig` variable, rather than calling `defaultComparePoint()` again.

Reply:
Right, introduced the variable to not use defaultComparePoint twice, and forgot to use it :)